package-lock.json

```diff
@@ -284,7 +284,7 @@
                 "yargs-parser": "^13.1.2"
             },
             "engines": {
-                "vscode": "^1.73.0"
+                "vscode": "^1.74.0-insider"
```

Comment:
Looks like package-lock still picked up the previous change. Just needs a regen I assume.